src/Corruptions.ts

```diff
@@ -232,6 +236,36 @@ const corruptionLoadoutSaveLoad = (save = true, loadout = 1) => {
     }
 }
 
+async function corruptionLoadoutGetNewName(loadout = 1) {
+    const maxChars = 9
+    let renamePrompt = await Prompt('What would you like to name Loadout ' + loadout + '? Names cannot be longer than ' + maxChars + ' characters. Nothing crazy!');
```

Comment:
```suggestion
    const renamePrompt = await Prompt(
        `What would you like to name Loadout ${loadout}? ` +
        `Names cannot be longer than ${maxChars} characters. Nothing crazy!`
    );
```
so it doesn't go off the page